chromadb/api/segment.py

```diff
@@ -248,7 +248,7 @@ def get_collection(
         if existing:
             return existing[0]
         else:
-            raise ValueError(f"Collection {name} does not exist.")
+            raise InvalidCollectionException(f"Collection {name} does not exist.")
```

Comment:
why this change

Reply:
`ValueError` results in a 500, which ends up as a generic `Exception` on the client side rather than a subclass of `ChromaError`. So this improves the API to return a 400 and makes it possible to write a test that can trigger a `ChromaError`.

Reply:
Oh woops, I was reading this diff on my phone and thought we went from InvalidCollectionException -> ValueError not other way around. This makes sense